server/src/main/java/org/apache/druid/server/coordination/ServerManager.java

```diff
@@ -293,8 +300,9 @@ public ServerManager(
         queryMetrics -> queryMetrics.segment(segmentIdString)
     );
 
-    CachingQueryRunner<T> cachingQueryRunner = new CachingQueryRunner<>(
+    QueryRunner<T> queryRunner = new CachingQueryRunner<>(
```

Comment:
Why is this change needed?

Reply:
will revert. must have been some temporary refactoring.